acknowledgements.md

```diff
@@ -5,33 +5,33 @@ The aim of this file is to acknowledge the software projects that have been used
 ## Castle.Core
 NSubstitute is built on the Castle.Core library, particularly Castle.DynamicProxy which is used for generating proxies for types and intercepting calls made to them so that NSubstitute can record them. 
 
-Castle.Core is maintained by the Castle Project [http://www.castleproject.org/] and is released under the Apache License, Version 2.0 [http://www.apache.org/licenses/LICENSE-2.0.html].
+Castle.Core is maintained by the Castle Project [https://www.castleproject.org/] and is released under the Apache License, Version 2.0 [https://www.apache.org/licenses/LICENSE-2.0.html].
 
 # Software used to help build NSubstitute
 
-## NUnit [http://www.nunit.org/]
-NUnit is used for coding and running unit and integration tests for NSubstitute. It is distributed under an open source zlib/libpng based license [http://www.opensource.org/licenses/zlib-license.html].
+## NUnit [https://nunit.org/]
+NUnit is used for coding and running unit and integration tests for NSubstitute. It is distributed under an open source zlib/libpng based license [https://www.opensource.org/licenses/zlib-license.html].
 
-## Rhino Mocks [http://www.ayende.com/projects/rhino-mocks.aspx]
-Used for mocking parts of the NSubstitute mocking library for testing. It is distributed under the BSD license [http://www.opensource.org/licenses/bsd-license.php].
+## Rhino Mocks [https://www.ayende.com/projects/rhino-mocks.aspx]
```

Comment:
This link is broken, maybe https://hibernatingrhinos.com/oss/rhino-mocks or https://github.com/hibernating-rhinos/rhino-mocks ?

Reply:
I think it's the first one

Reply:
changed to https://hibernatingrhinos.com/oss/rhino-mocks